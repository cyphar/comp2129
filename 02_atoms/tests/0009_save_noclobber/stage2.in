LOAD save.as
PLAYFROM 0
SAVE save.as
