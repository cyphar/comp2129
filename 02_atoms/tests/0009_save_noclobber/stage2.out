Game Loaded

Game Ready
Red's Turn

File Already Exists

